ruka/cnpg/cnpg.sh

```diff
@@ -77,28 +77,28 @@ spec:
       secret:
         name: cnpg-cluster-app-user
 
-  backup:
-    barmanObjectStore:
-      destinationPath: ${AWS_ACCESS_BUCKET}
-      s3Credentials:
-        accessKeyId:
-          name: cnpg-aws-creds
-          key: ACCESS_KEY_ID
-        secretAccessKey:
-          name: cnpg-aws-creds
-          key: ACCESS_SECRET_KEY
-      wal:
-          compression: gzip
-
-    retentionPolicy: "90d"
+#  backup:
```

Comment:
Why is the backup block being commented out instead of deleted?

Reply:
because i don't want to lose the block so i can test later on the normal backups when other version come out, but on the TTS and Summit deployment the backup block will be gone.

Reply:
OK. My general preference is to delete dead configuration / code.